tests/test_routes.py

```diff
@@ -18,10 +18,8 @@
 TestOrder API Service Test Suite
 """
 
-# pylint: disable=duplicate-code
```

Comment:
curious if this was necessary for anything or not? (Even though it was commented)